web-sdk.js

```diff
@@ -1876,8 +1871,14 @@
                 i += "; expires=" + o.toGMTString();
             }
             "path" in n && (i += "; path=" + n.path);
+            "domain" in n && (i += "; domain=" + n.domain);
             n.secure && (i += "; secure");
-            m.document.cookie = i;
+            // m.document.cookie = i;
+            if (t) {
+              localStorage.setItem(e, t);
+            } else {
+              localStorage.removeItem(e);
```

Comment:
Original logic doesn't seem to do this?

Also the key (`e`) and val (`t`) are run through the `f` function first; is `f` equal `encodeURIComponent`?